errors_test.go

```diff
@@ -39,6 +37,18 @@ func TestErrorHandler(t *testing.T) {
 		require.Equal(t, http.StatusNotFound, errResponse.(HTTPError).StatusCode())
 	})
 
+	t.Run("not duplicate HTTPError", func(t *testing.T) {
+		err := HTTPError{
+			Err: errors.New("HTTPError"),
+		}
+		errResponse := ErrorHandler(err)
+
+		var httpError HTTPError
+		require.ErrorAs(t, errResponse, &httpError)
+		require.False(t, errors.As(httpError.Err, &HTTPError{}))
```

Comment:
You can use this if you bump testify to latest version. This missing helper was added

```suggestion
		require.NotErrorAs(t, httpError.Err, &HTTPError{})
```

Reply:
Ah thanks. I only saw .NotErrorIs and just assumed they didn’t do it.